openmetadata-ui/src/main/resources/ui/src/components/common/facetfilter/FilterContainer.tsx

```diff
@@ -42,7 +42,7 @@ const FilterContainer: FunctionComponent<FilterContainerProp> = ({
 
   return (
     <label
-      className="filter-group tw-justify-between tw-mb-2.5 tw-cursor-pointer"
+      className="filter-group tw-justify-between m-b-xs tw-cursor-pointer"
```

Comment:
Can we remove tailwind styling from here and use existing styles?

Reply:
@aniketkatkar97 updated!